Packs/Cybereason/Scripts/CybereasonPreProcessingExample/CybereasonPreProcessingExample.yml

```diff
@@ -0,0 +1,22 @@
+commonfields:
+  id: CybereasonPreProcessingExample
+  version: -1
+name: CybereasonPreProcessingExample
+script: ''
+type: python
+subtype: python3
+tags:
+- preProcessing
+- Cybereason
+- example
+comment: |-
+  Preprocessing script to run when fetching Cybereason malops.
+  Will check if malop was already fetched, and will then update the existing incident, otherwise will create a new incident.
+enabled: true
+scripttarget: 0
+runonce: false
+runas: DBotWeakRole
```

Comment:
@idovandijk , When using the DbotWeakRole we had issues with pre-process rules. Do you know if that was solved?

Reply:
I think Aviad had fixed the issue but also not sure